code/datums/uplink_item.dm

```diff
@@ -219,6 +219,14 @@ var/list/uplink_items = list()
 	gamemodes = list(/datum/game_mode/nuclear)
 	surplus = 35
 
+/datum/uplink_item/dangerous/bioterrorfoam
+	name = "Crowd control chemical foam grenade"
+	desc = "A powerful chemical foam grenade which creates a deadly torrent of foam that will mute, blind, confuse, mutate, and irritate carbon lifeforms. Specially brewed by Tiger Cooperative chemical weapons specialists using additional spore toxin. Ensure suit is sealed before use."
+	item = /obj/item/weapon/grenade/chem_grenade/bioterrorfoam
+	cost = 5
+	gamemodes = list(/datum/game_mode/nuclear,/datum/game_mode/gang)
```

Comment:
I was definitely drunk when I decided to add it to hang. But I think the reasoning was something along the lines of "keep people away from the dominator" and "get people away from their dominator" kind of scenario.

In other words, purely crowd control.